mobile/status.go

```diff
@@ -280,6 +282,12 @@ func CreateAccountAndLogin(requestJSON string) string {
 
 	api.RunAsync(func() error {
 		log.Debug("starting a node and creating config")
+		var now uint64
+		if now, err = timesource.GetCurrentTimeInMillis(); err != nil {
+			log.Error("CreateAccountAndLogin. failed to get current time in millis", "error", err)
+			return err
+		}
+		request.CustomizationColorUpdatedAt = now
```

Comment:
I don't think this is the right place for updating the customization color, it should go in `CreateAccountAndLogin` and not sure it should be part of the request, since it's not user controlled?